Mariuzzo.Do.DataAnnotations.Test/CedulaAttributeTest.cs

```diff
@@ -29,5 +31,78 @@ public void TestInvalidCedulas()
             Assert.IsFalse(attr.IsValid("\n"));
             Assert.IsFalse(attr.IsValid("\n \t"));
         }
+        [TestMethod]
+        public void TestValidCedulasExceptionsShouldReturnValidIfIgnoreExceptionsIsFalse()
+        {
+            var attr = new CedulaAttribute { IgnoreExceptions = false };
+            _validExceptions.ForEach(c => Assert.IsTrue(attr.IsValid(c)));
```

Comment:
Right now this is fine as we're using MSTest and we don't have any extension libraries for tests, but eventually we should avoid executing loops directly in the test and use assertions that compare lists.